app/main/views/buyers.py

```diff
@@ -263,8 +263,16 @@ def seller_feedback_email(brief_id):
 @role_required('admin')
 def find_team_by_team_id():
     team_id = request.args.get('team_id')
+    if team_id:
+        team_info = data_api_client.req.admin().team(team_id).get()
```

Comment:
what happens when team_info is null?

Reply:
fixed

Reply:
now an error shows when you just click search....
also it is normally bad practice to have a catch all exception handler.